actions/submit-signing-request/task.ts

```diff
@@ -264,8 +264,8 @@ export class Task {
         finally {
 
             core.debug(`Deleting temp directory ${tmpDir}`);
-            // delete the temp DIR
-            fs.unlinkSync(tmpDir);
+            // delete the temp ZIP
+            fs.unlinkSync(tmpZipFile);
```

Comment:
Yes, it should work to remove the entire directory. Just not with `unlinkSync`, but with `rmSync`.

Background:
https://www.geeksforgeeks.org/node-js-fs-unlinksync-method/
> This function does not work on directories, therefore it is recommended to use fs.rmdir() to remove a directory.

However, to remove recursively (including files), https://nodejs.org/api/fs.html#fsrmdirsyncpath-options says
> To get a behavior similar to the `rm -rf` Unix command, use fs.rmSync() with options { recursive: true, force: true }.

Please let's try that to properly clean up

Reply:
As discussed, I use the runner TMP directory (RUNNER_TEMP) and do not remove the temporary file. According to GitHub Actions documentation, this folder is cleaned after each job run.